roles/edpm_ovn/tasks/configure.yml

```diff
@@ -14,6 +14,25 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Apply ovsdb configuration from configMaps
+  block:
+  - name: Check that config file exists
+    stat:
+      path: "{{ edpm_ovn_config_src }}/ovsdb-config"
+    register: ovsdb_config_exists
+
+  - name: Load ovsdb configuration from file
+    ansible.builtin.include_vars:
+      file: "{{ edpm_ovn_config_src }}/ovsdb-config"
+      name: edpm_ovn_configmap_facts
+    delegate_to: localhost
+    when:
+    - ovsdb_config_exists
```

Comment:
this check is invalid, I should check the "exists": true here. That's why this task is executed in CI even when the previous one couldn't find the file.

Reply:
(I should add a scenario for missing file)

Reply:
Added new scenario for missing config file.